tests/HttpClientTest.php

```diff
@@ -12,22 +12,22 @@
 // the License.
 
 use Meplato\Store2\HttpClient;
+use PHPUnit\Framework\Attributes\Group;
 
 /**
  * Tests the Meplato Store 2 HTTP client.
  */
-class HttpClientTest extends BaseTest
+class HttpClientTest extends BaseTestCase
 {
 	/**
 	 * Test if the HTTP client works, in general.
-	 *
-	 * @group root
-	 * @group http
 	 */
+	#[Group('root')]
+	#[Group('http')]
 	public function testNewHttpClient()
 	{
 		$client = new HttpClient();
-		$resp = $client->execute("GET", "https://store2.meplato.com/");
+		$resp = $client->execute("GET", "https://store.meplato.com/");
```

Comment:
This test is failing when the store is down/in maintenance
I do not know how to feel about that.